jormungandr-integration-tests/src/jcli/key/from_bytes.rs

```diff
@@ -52,6 +52,6 @@ pub fn test_from_bytes_for_unknown_key() {
     jcli_wrapper::assert_key_from_bytes_fails(
         &byte_key_file,
         "ed25519Exten",
-        "Invalid value for '--type <key_type>'",
+        "Invalid value for '--type <key-type>':",
```

Comment:
Command renaming behavior [changed here](https://github.com/TeXitoi/structopt/blob/master/CHANGELOG.md#change-default-case-from-verbatim-into-kebab-by-0ndorio-202)

Reply:
this one is not a problem, it does not change the user's APIs